(DOCSP-14328, DOCSP-14328) Android Object Models & Schemas, Versions & Migration (#759)
mongodb/docs-realm: source/sdk/android/fundamentals/schema-versions-and-migrations.txt

   :local:
   :backlinks: none
   :depth: 2
   :class: singlecol

.. _android-schema-version:

Schema Version
--------------

A **schema version** identifies the state of a :ref:`{+backend-schema+}
<android-realm-schema>` at some point in time. {+client-database+} tracks the schema
version of each {+realm+} and uses it to map the objects in each {+realm+}
to the correct schema.

> Schema versions are ascending integers that you can optionally include

Reviewer: ```suggestion
Schema versions are ascending integers that you may include
```

> Schema versions are ascending integers that you can optionally include

Reviewer: Nitpick: Strictly speaking, is this true? I know a new version must be higher than the last, but is the integer itself ascending, per se? This information is also contained in the monotonic increment section.
```suggestion
Schema versions are integers that you can optionally include
```
